arkouda/numeric.py

```diff
@@ -241,28 +295,34 @@ def where(condition : pdarray, A : Union[Union[int,float], pdarray],
                         B : Union[Union[int,float], pdarray]) -> pdarray:
     """
     Returns an array with elements chosen from A and B based upon a 
-    conditioning array.
+    conditioning array. As is the case with numpy.where, the return array
+    consists of values from the first array (A) where the conditioning array 
+    elements are True and from the second array (B) where the conditioning
+    array elements are False.
     
     Parameters
     ----------
     condition : pdarray
         Used to choose values from A or B
     A : scalar or pdarray
-        Value(s) used when condition is True
+        Value(s) used when condition is True for A
```

Comment:
I don't understand the addition of "for A" here (and in line 310) -- is it meant to clarify something?

Reply:
Gotcha, agreed, the addition is unneeded, removed.